src/api/XR.js

```diff
@@ -55,6 +71,11 @@ export default class XR extends EventTarget {
       await this[PRIVATE].devicePromise;
     }
 
+    if (!XRSessionModes.includes(mode)) {
```

Comment:
I believe this was previously in here, although removed so that it's possible to extend the polyfill with a custom mode supported by a custom XRDevice, like when prototyping `immersive-ar`

Reply:
This is the same as comment here, not sure if the latest has been updated: https://github.com/immersive-web/webxr-polyfill/pull/95#issuecomment-538675914